src/app/collection/config/collection-config.component.html

```diff
@@ -2,8 +2,8 @@
      [style.opacity]="collection ? '1' : '0'"
      *ngIf="collection">
   <div class="d-flex flex-row justify-content-start">
-    <a class="fas fa-chevron-left bigger mr-2 text-success" (click)="goToCollectionsPage()"></a>
-    <i class="color-class fa-fw {{ collection.icon }} collection-icon clickable"
+    <i class="fas fa-chevron-left bigger mr-3 clickable" (click)="goToCollectionsPage()"></i>
```

Comment:
Where does this `clickable` class come from and why do we use it everywhere?

Reply:
It is defined in `common.scss` and has our default behaviour of clickable items (green colour with darker green on hover)